woocommerce/woocommerce-ios
# Shipping Label: Refactor ReprintShippingLabel for use printing new shipping labels

Part of #4089

## Description

This refactors the `Print Shipping Label` UI (view controller, view model, and coordinator), renaming the classes and methods to make it clearer to use in the shipping label purchase flow (not just for reprinting existing shipping labels).

A followup PR will include the changes to customize that screen for use in the shipping label purchase flow.

## Changes

* Renamed classes, methods, etc. to use "print" instead of "reprint" everywhere, unless it's specifically about reprinting.
* No visual changes.

## Testing

1. Make sure you have configured the WooCommerce WCShip plugin, and you have one or more orders with a purchased shipping label.
2. Open an order detail that has an existing shipping label.
3. Tap the "Print Shipping Label" button and confirm the "Print Shipping Label" screen appears.
4. Confirm you can still interact with this screen as expected (change the paper size, tap the help text, print the label).

## Submitter Checklist

Update release notes:
